File: examples/id/generate/node/generateABigflakeId.js
const { IdService } = require("m3o/id");

// Generate a unique ID. Defaults to uuid.
async function generateAbigflakeId() {
  let idService = new IdService(process.env.MICRO_API_TOKEN);
  let rsp = await idService.generate({
    type: "bigflake",
  });
  console.log(rsp);
}

generateAbigflakeId();

File: examples/go/id/generate/generateABigflakeId.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/id"
)

// Generate a unique ID. Defaults to uuid.
func GenerateAbigflakeId() {
	idService := id.NewIdService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := idService.Generate(&id.GenerateRequest{
		Type: "bigflake",
	})
	fmt.Println(rsp, err)
}
